cypress/integration/features/editSnapshot.spec.js

```diff
@@ -100,4 +97,16 @@ context('Edit snapshot', () => {
         .and('contain', 'Note');
     });
   });
+
+  describe('Back button', () => {
+    it('Sends the user back to Single View', () => {
+      cy.get('[data-testid=back-link-test]')
+        .should('contain', 'Back to Single View')
+        .and(
+          'have.attr',
+          'href',
+          'https://staging-singleview.hackney.gov.uk/customers/wub/view'
```

Comment:
Looks good. Are we only saving SV ID in system IDs and if so why is it an array? Just a general question

Reply:
Nah we're saving multiple ids